bundler/lib/bundler/definition.rb

```diff
@@ -517,7 +517,30 @@ def materialize(dependencies)
     end
 
     def precompute_source_requirements_for_indirect_dependencies?
-      @remote && sources.non_global_rubygems_sources.all?(&:dependency_api_available?) && !sources.aggregate_global_source?
+      precompute = true
+      reasons = []
+
+      if @remote
+        unless sources.non_global_rubygems_sources.all?(&:dependency_api_available?)
+          reasons << "any of the involved gem servers does not implement a dependency API"
```

Comment:
OK. I will change the message printing the servers not implementing dependency APIs. I would like to use the "do not" rather than "don't", because the "do not" is more often used in the source code. It seems the "don't" is used in the source comment.

```
$ grep -r " do not " bundler/lib/ | wc -l
41

$ grep -r " don't " bundler/lib/ | wc -l
36
```